src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -1064,7 +1064,7 @@ namespace EIRPlantLoopHeatPumps {
                                         "; entered curve name: " + capEIRtName.get<std::string>());
                         errorsFound = true;
                     }
-                    auto &capEIRplrName = fields.at("electric_input_to_output_ratio_modifier_function_of_temperature_curve_name");
+                    auto &capEIRplrName = fields.at("electric_input_to_output_ratio_modifier_function_of_part_load_ratio_curve_name");
```

Comment:
Good catch. I'd take the opportunity to. rename this variable and the ones next to it as appropriate to be correct by themselves.
This `capEIRplrName` should be `eirFplrName` or something. Above `capEIRtName` should be `eirFtName`. ("Cap" shouldn't be part of the name, since it's not a capacity modifier curve).

Reply:
This has been addressed in the latest commit.

Reply:
Great. I guess a unit test is needed too.

Reply:
Ready to go here.